leaderboard/enriching/enricher_cache.go

```diff
@@ -0,0 +1,111 @@
+package enriching
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"github.com/go-redis/redis/v8"
+	"github.com/topfreegames/podium/leaderboard/v2/model"
+	"go.uber.org/zap"
+	"time"
+)
+
+// cacheKeyFormat is {tenantID}:{leaderboardID}:{memberID}
+const cacheKeyFormat = "leaderboards-enrich-caching:%s:%s:%s"
+
+type enricherCache struct {
```

Comment:
I would change this name to `redisEnricherCache` or `redisBackedEnricherCache`. The same for the constructor, `NewRedisEnricherCache` or `NewRedisBackedEnricherCache`.

Reply:
Renamed.